acados/ocp_nlp/ocp_nlp_common.c

```diff
@@ -146,8 +146,7 @@ ocp_nlp_config *ocp_nlp_config_assign(int N, void *raw_memory)
  * dims
  ************************************************/
 
-// TODO(oj): should be static, but used by current ocp_nlp c++ interface
-int ocp_nlp_dims_calculate_size_self(int N)
+static int ocp_nlp_dims_calculate_size_self(int N)
```

Comment:
The reason it was there, was that Robin needed in the C++ interface. If not it is not used any longer in there, then we can just merge it again with `dims_calculate_size`.

Reply:
The C++ interface cannot be used with recent acados versions anyway.
So, I just wanted to make it clear it should just be used internally.
Merging with non `self` ones can still be done later.